code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -12,6 +12,10 @@
 /mob/living/carbon/human/slip(knockdown_amount, obj/O, lube, paralyze, forcedrop)
 	if(HAS_TRAIT(src, TRAIT_NOSLIPALL))
 		return FALSE
+	if(shoes && isclothing(shoes))
+		var/obj/item/clothing/CS = shoes
+		if (CS.clothing_flags & NOSLIP_ALL)
```

Comment:
This should probably also be added a couple lines down in `experience_pressure_difference` like the normal NOSLIP so pressure doesn't knock people away.